src/practices/JavaScript/ESLintWithoutErrorsPractice.ts

```diff
@@ -92,25 +92,33 @@ export class ESLintWithoutErrorsPractice extends PracticeBase {
       }
 
       try {
-        // eslint-disable-next-line @typescript-eslint/no-require-imports
-        baseConfig = require(nodePath.resolve(ctx.fileInspector.basePath!, eslintConfig[0].path));
+        //load config file according to its extension
+        if (eslintConfig[0].extension === '.yml' || eslintConfig[0].extension === '.yaml') {
+          baseConfig = yaml.safeLoad(await ctx.fileInspector.readFile(eslintConfig[0].path));
```

Comment:
```suggestion
          baseConfig = yaml.load(await ctx.fileInspector.readFile(eslintConfig[0].path));
```
the new version of the package yaml deprecated `safeLoad()`